### PR TITLE
Improve PushNotificationIOS documentation

### PR DESCRIPTION
**Pull request by respectTheCode**
The documentation currently list a few but not all of the steps required. This changes completes that list of required project changes.

This addresses #5612

### CODE REVIEW

File: Libraries/PushNotificationIOS/PushNotificationIOS.js [hunk 1]
```diff
@@ -36,9 +36,10 @@ const DEVICE_LOCAL_NOTIF_EVENT = 'localNotificationReceived';
  *
  * [Manually link](docs/linking-libraries-ios.html#manual-linking) the PushNotificationIOS library
  *
- * - Be sure to add the following to your `Header Search Paths`:
- * `$(SRCROOT)/../node_modules/react-native/Libraries/PushNotificationIOS`
- * - Set the search to `recursive`
+ * - Add the follwoing to your Project: `node_modules/react-native/Libraries/PushNotificationIOS/RCTPushNotification.xcodeproj`
```
Comment (hramos) [inline Libraries/PushNotificationIOS/PushNotificationIOS.js]: "Following" is misspelled.

File: Libraries/PushNotificationIOS/PushNotificationIOS.js [hunk 2]
```diff
@@ -36,9 +36,10 @@ const DEVICE_LOCAL_NOTIF_EVENT = 'localNotificationReceived';
...
+ * - Add the follwing to `Link Binary With Libraries`: `libRCTPushNotification.a`
```
Comment (hramos) [inline Libraries/PushNotificationIOS/PushNotificationIOS.js]: "Following" is misspelled here as well.

**Comment by facebook-github-bot-4** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by facebook-github-bot-4** [comment]
@respectTheCode updated the pull request.

**Comment by nicklockwood** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-5** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1808743359354371/int_phab) to review.

**Pull request closed without merging**